backend/app/mailers/enrollment_reminder_mailer.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+class EnrollmentReminderMailer < ActionMailer::Base
+  layout false
+
+  def reminder_draft_enrollment_email
+    @target_api_label = data_provider_config["label"]
+    @enrollment = Enrollment.find(params[:enrollment_id])
+    @demandeur_given_name = @enrollment.demandeurs.first.given_name
+    @demadeur_family_name = @enrollment.demandeurs.first.family_name
+    @url = "#{ENV.fetch("FRONT_HOST")}/#{params[:target_api].tr("_", "-")}/#{params[:enrollment_id]}"
+
+    mail(
+      to: enrollment.demandeurs.pluck(:email),
```

Comment:
when I run the job I got an error:

```
2022-12-22T06:34:53.540Z pid=45477 tid=13ft WARN: NameError: undefined local variable or method `enrollment' for #<EnrollmentReminderMailer:0x0000000000ec90>
Did you mean?  @enrollment
2022-12-22T06:34:53.540Z pid=45477 tid=13ft WARN: /opt/apps/datapass-backend/current/app/mailers/enrollment_reminder_mailer.rb:14:in `reminder_draft_enrollment_email'
```

plz write a test that throw this error before fixing it.

Reply:
Thank you for fixing the rebase problems. 

I've added some little changes to the test, specially about the  `Timecop.freeze(Time.now.change(year: 2022, month: 12))` to add a fix date.